### PR TITLE
Fix error added by qbittorrent 5.0 pymedusa/Medusa#11844

### PR DESCRIPTION
**Pull request by borntohonk**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

Proposed fixes for error(s) from https://github.com/pymedusa/Medusa/issues/11844 caused by qbittorrent 5.0 api changes, which for reasons unknown to mortal man is that they changed "pause/paused" to "stop/stopped", and "resume/resumed" to "start".

Things addressed/fixed:
* Download handler is unable to process completed & "stopped" torrents (completed & uploading/seeding worked fine), which affects people who don't seed or if the download handlers schedule is slower than seeding goal is hit.

* Log-level error spawned from not being able to parse this new unknown "stopped" state.

Also tested:
qQbittorrent v4.6.7, to ensure new changes to accommodate qQbittorrent v5.0+ does not negatively affect previous version(s)

### CODE REVIEW

File: medusa/schedulers/download_handler.py [hunk 1]
```diff
@@ -45,6 +45,7 @@ class ClientStatusEnum(Enum):
 
     SNATCHED = 0  # 0
     PAUSED = 1  # 1
+    STOPPED = 1
```
Comment (p0psicles) [inline medusa/schedulers/download_handler.py:48]: the only thing i'm not sure here, is if we need another download handler state. What benefit brings this?

Comment (borntohonk) [reply medusa/schedulers/download_handler.py:48]: Semantics to go along with qbittorent deciding to rebrand pause to stop, and the other parts adding "stop" wouldn't work was my interpretation.

Comment (borntohonk) [reply medusa/schedulers/download_handler.py:48]: Also a quick effort to not affect other torrent programs that don't conform to whatever qbittorrent maintainers silly decision, and retain backwards compatibility with pre 5.0.0 qbittorrent versions.

at least that was my intention

Comment (borntohonk) [reply medusa/schedulers/download_handler.py:48]: @p0psicles i remember now
https://github.com/pymedusa/Medusa/pull/11857/files#diff-28c840b0e4fc446765e43a85e5cb6aed6a874eee15753e980007e12c86fa6096R243
This line would not work unless a new state of stopped was defined, and i didn't want to cause any unwanted effects to existing ecosystem, only support qbittorrent 5.0+

Comment (borntohonk) [reply medusa/schedulers/download_handler.py:48]: (i am first now registering i made a typo in that function too)

Comment (borntohonk) [reply medusa/schedulers/download_handler.py:48]: i summarized this better here: https://github.com/pymedusa/Medusa/issues/11844#issuecomment-2395316482

Comment (borntohonk) [reply medusa/schedulers/download_handler.py]: I had time to deploy this on a physical machine again, and removed the changes to download_handler.py

**Comment by XxUnkn0wnxX** [comment]
Is this still being worked on, or is it stuck in limbo?

**Comment by borntohonk** [comment]
> Is this still being worked on, or is it stuck in limbo?

I'm not a maintainer so that's not up to me to decide, but I did remove the downloader_handler state per comment, and the "fix" itself does do what it's supposed to do.

**Comment by borntohonk** [comment]
(ideally, one would go nag qbittorrent folks to make it backwards compatible instead of this new nonsense https://github.com/qbittorrent/qBittorrent/blob/master/src/webui/www/private/scripts/mocha-init.js#L523-L549 )

**Review by medariox — DISMISSED** [review]

**Comment by medariox** [comment]
LGTM! Thank you 🙏

**Pull request merged**